### PR TITLE
Hit the API

### PR DESCRIPTION
**Pull request by sunga12**
- Implemented the "Refresh" button that gets all scores for a game created by me from the API
- Implemented the form "Submit" button that saves a score for the game 
- Used arrow functions instead of the function keyword.
- Used async and await JavaScript features to consume the API.

### CODE REVIEW

File: src/postData.js [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+  const response = await fetch('https://us-central1-js-capstone-backend.cloudfunctions.net/api/games/RP7P3DgAooG12Jvz9hXa/scores', {
+    method: 'POST',
+    headers: {
+      'Content-Type': 'application/json',
+    },
+    body: JSON.stringify(data),
+  });
+
+  return response.json();
+}
```
Comment (AbelG101) [inline src/postData.js]: - [ ] Ouch! this is really close but once user data is submitted it's not reflected on the scoreboard, the refresh button has to be clicked to see it on the scoreboard. This leads to bad UX as the user might think their score wasn't added properly so please make sure to fix this. ✔
  firefox_3PxMIwIOBc

**Review by AbelG101 — CHANGES_REQUESTED** [review]
# Hi @sunga12 👋, 

You're doing a good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights
- Correct use of GitHub Flow ✔
-  Well-structured files ✔
- All linter checks are passing ✔
- Descriptive PR title with a great summary ✔

## Required Changes ♻️
- _Check the comments under the review._
- you just have to make a few changes and you're on your way 🚀.


## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better._

You can also consider:
- N/A

Cheers, and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/postData.js (hunk 1 above)
Comment (sunga12) [reply src/postData.js]: Hi, @AbelG101, This was done according to the specifications given [here](https://github.com/microverseinc/curriculum-javascript/blob/main/leaderboard/m2_send_receive_data.md). That only when the refresh button is clicked that it when the scoreboard shows the updated scores with the new user score.

**Review by TadesseDev — APPROVED** [review]
### Hi @sunga12 

### I'm Tadesse Alemayehu your reviewer for this PR. :hand: 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 
giphy (1)



## Highlights
 - [x] Good job making the changes requested by the previous code reviewer. ✅

## Optional suggestions

- N/A

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**